examples/basics/quick_start.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    
</pre><p>Don't use the default integration if not relevant</p>

<pre class="ql-syntax" spellcheck="false">dataset = client.create_dataset(name="Quick Start Example Dataset", iam_integration=None)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>